packages/example/src/components/Transfer/Transfer.tsx

```diff
@@ -1,21 +1,53 @@
-import React from "react";
+import React, {useState} from "react";
 import {Box, Button, Card, CardContent, CardHeader, Grid, TextField} from '@material-ui/core/';
+import {getInjectedMetamaskExtension} from "../../services/metamask";
+
+interface ITransferProps {
+    network: string
+}
+
+export const Transfer: React.FC<ITransferProps> = ({network}) => {
+    const [recipient, setRecipient] = useState<string>("");
+    const [amount, setAmount] = useState<string | number>("");
+
+    const handleCurrency = (network: string): string => {
+        switch(network) {
+            case "kusama": return "KSM";
+            case "westend": return "WST";
+            default: return ""
+        }
+    }
+
+    const handleRecipientChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setRecipient(event.target.value);
+      };
+    const handleAmountChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setAmount(event.target.value);
+    };
+
+    const onSubmit = async () => {
+        const provider = await getInjectedMetamaskExtension();
+        
+        if(provider) {
+            const api = await provider.getMetamaskSnapApi();
+            const signedTx = await api.sign(amount, recipient);
+        }
+    };
 
-export const Transfer = () => {
     return (
         <Card>
             <CardContent>
                 <CardHeader title="Transfer"/>
                 <Grid container alignItems="center" justify="space-between">
                     <Grid item xs={12}>
-                        <TextField size="medium" fullWidth id="recipient" label="Recipient" variant="outlined" />
+                        <TextField onChange={handleRecipientChange} size="medium" fullWidth id="recipient" label="Recipient" variant="outlined" />
                         <Box m="0.5rem"/>
-                        <TextField size="medium" fullWidth id="recipient" label="Amount" variant="outlined" />
+                        <TextField placeholder={handleCurrency(network)} onChange={handleAmountChange} size="medium" fullWidth id="recipient" label="Amount" variant="outlined" />
```

Comment:
I don't think currency should be placeholder. Use this:
https://material-ui.com/components/text-fields/#input-adornments